app/Repositories/PhotoRepository.php

```diff
@@ -44,7 +44,11 @@ public function create(array $data, $signupId)
     {
         $postEvent = Event::create($data);
 
-        $fileUrl = $this->aws->storeImage($data['file'], $data['campaign_id']);
+        if (array_key_exists('campaign_id', $data)) {
+            $fileUrl = $this->aws->storeImage($data['file'], $data['campaign_id']);
+        } else {
+            $fileUrl = $this->aws->storeImage($data['file'], $signupId);
+        }
```

Comment:
@sbsmith86 I wanted to make sure that this part looks okay, I'm not sure if it matters what the file name is? Since that's all the `campaign_id` is used for it made sense to me to not go out of our way to pass the `campaign_id` when we wouldn't otherwise